docs/adr/2023-09-08_026-network-resilience.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: 26
+title: |
+  26. Network failures model
+authors: []
+tags: [Proposed]
+---
+
+## Status
+
+Proposed
```

Comment:
I think we call it `Draft`, but I also like `Proposed` (let's change all to that status if you want that)

Reply:
```suggestion
Draft
```